controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -531,6 +532,19 @@ func (r *DataScienceClusterReconciler) SetupWithManager(ctx context.Context, mgr
 			}),
 			builder.WithPredicates(argoWorkflowCRDPredicates),
 		).
+		Watches( // ingresscontroller
```

Comment:
do we need this watch ? it seems the same enqueue logic is performed by the` Watches(corev1.Secret)` below

Reply:
all in all, why we need this IngressController? feel it was no need added there at first